include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -63,55 +65,35 @@ namespace aspect
            * SimulatorAccess is not initialized and only Rank 0 should give
```

Comment:
I think this is not longer true as well. We now have SimulatorAccess during the call to GPlates::initialize(). Remove the second sentence.